Cython/Includes/cpython/datetime.pxd

```diff
@@ -90,9 +90,9 @@ cdef extern from "datetime.h":
     int PyDateTime_TIME_GET_MICROSECOND(object o)
 
     # Getters for timedelta (C macros).
-    int PyDateTime_DELTA_GET_DAYS(object o)
-    int PyDateTime_DELTA_GET_SECONDS(object o)
-    int PyDateTime_DELTA_GET_MICROSECONDS(object o)
+    int PyDateTime_DELTA_GET_DAYS(timedelta o)
+    int PyDateTime_DELTA_GET_SECONDS(timedelta o)
+    int PyDateTime_DELTA_GET_MICROSECONDS(timedelta o)
```

Comment:
Not a good idea. This will enforce type checks on caller side if the type is not known, which will either reduce the performance or make users uglify their input with type casts.

These are clearly macros, designed for speed. Let it crash if users misapply them.

Reply:
there's not a performance hit to having it be object?